nilrt-build-init.env

```diff
@@ -2,6 +2,21 @@
 
 SCRIPT_ROOT=$(realpath $(dirname ${BASH_SOURCE:-${0}}))
 
+enable_ni_org_conf=false
+positionals=()
+
+while [ $# -ge 1 ]; do case "$1" in
+	-o|--org)
+		enable_ni_org_conf=true
```

Comment:
I'm torn, the selfish side of me wants our use case to be the "no parameters - just add in the NI stuff".  However I also recognize that users will "always" fail if they just invoke the script without parameters.  So I understand why you did the logic the way you did.

Reply:
I figure NI developers are smart enough to know how to alias or wrap the script in their workspace to automatically use `--org`.